src/containers/BooksForm.js

```diff
@@ -0,0 +1,54 @@
+/* eslint-disable react/prop-types */
+/* eslint-disable no-unused-vars */
+import React, { useState } from 'react';
+import { connect } from 'react-redux';
+import { createBook } from '../actions/index';
+
+const BooksForm = ({ dispatch }) => {
+  const [state, setState] = useState({
+    id: Date.now(),
+    title: '',
+    category: 'Action',
+  });
+  const handleChange = (e) => {
+    if (e.target.name === 'title') {
+      setState({
+        ...state, title: e.target.value,
+      });
+    }
+    if (e.target.name === 'category') {
+      setState({
+        ...state, title: e.target.value,
+      });
+    }
+  };
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    setState({
+      ...state, id: Date.now(),
+    });
+    dispatch(createBook(state));
+  };
```

Comment:
```
const mapDispatchToProps = (dispatch) => ({
  addBook: (book) => {
    dispatch(createBook(book));
  },
});

export default connect(null, mapDispatchToProps)(BooksForm);
```

 Implement createbook in mapDispatchToProps, then as a prop, call the function in the handleSubmit function.